README.md

```diff
@@ -59,9 +59,19 @@ After running the installation script, NErlNet directory can be accessed via the
 ## Python API and Jupyter: 
 
 1. Create and activate a virtual environment for Nerlnet https://docs.python.org/3/library/venv.html
-2. Install required modules ```pip install -r src_py/requirements.txt```
+2. Install required modules ```pip install -r src_py/reqCleaned.txt```
 3. Install Jupyter Notebook ```pip install jupyterlab```
 4. import the ApiServer module to the notebook. 
 5. Follow the example: https://github.com/leondavi/NErlNet/blob/master/src_py/Deprecated/flow_example.ipynb 
 
+## Running remotely on azure 
```

Comment:
Remove all the part of running remotely on azure from the readme. 
This is excellent that you wrote it. 
Send me a word document of it. This is only for personal use.

Reply:
Fixed